test/python/utils.py

```diff
@@ -0,0 +1,15 @@
+import xarray as xr
+from math import floor, log10
+
+
+def pfbs_are_equal_to_n_sig_figs(pfb1, pfb2):
```

Comment:
This function support writing the same type of assertion as the TCL where we check that the run output is equal to the expected value within some tolerance